package.json

```diff
@@ -6,9 +6,7 @@
   "workspaces": {
     "packages": [
       "packages/*",
-      "examples/monaco-graphql-webpack",
-      "examples/monaco-graphql-nextjs",
-      "examples/monaco-graphql-react-vite"
```

Comment:
I added it to test removing unused languages and avoid regressions

Reply:
no worries, this PR will take a long while before it will be stable enough for review even! I expect to do a significant rebase next week